python-client/giskard/ml_worker/testing/registry/giskard_test.py

```diff
@@ -95,7 +95,7 @@ def get_builder(self):
 
 
 class GiskardTestMethod(GiskardTest):
-    params = None
+    params = {}
```

Comment:
Maybe `params = dict()` to be more explicit, because sometimes `{}` could be confused with a `set`.